.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
     "eslint.alwaysShowStatus": true,
-    "eslint.lintTask.enable": true
+    "eslint.lintTask.enable": true,
```

Comment:
Please undo this change. Not only is this change unnecessary, it also makes this file invalid JSON.